plugins/obs-ffmpeg/texture-amf.cpp

```diff
@@ -1323,12 +1391,14 @@ static bool amf_avc_init(void *data, obs_data_t *settings)
 	int rc = get_avc_rate_control(rc_str);
 
 	set_avc_property(enc, RATE_CONTROL_METHOD, rc);
-	if (rc != AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_CONSTANT_QP)
+	if (rc != AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_CONSTANT_QP &&
+	    rc != AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_HIGH_QUALITY_VBR &&
+	    rc != AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_HIGH_QUALITY_CBR) {
 		set_avc_property(enc, ENABLE_VBAQ, true);
+	}
```

Comment:
change the  "&&" to "||" to the VBAQ work only in rc"CQP or HQCBR or CBR  #"

set_avc_property(enc, RATE_CONTROL_METHOD, rc);
	if (rc == AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_CONSTANT_QP ||
	    rc == AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_HIGH_QUALITY_VBR ||
	    rc == AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD_HIGH_QUALITY_CBR) {
		set_avc_property(enc, ENABLE_VBAQ, true);
	}